Core/src/IfcGeometryConverter/SplineConverter.h

```diff
@@ -147,12 +147,149 @@ namespace OpenInfraPlatform
 					 * \internal	The Code of the function is in the commented out part at the end of the file.
 					 */
 					void convertIfcBSplineSurface(
-						const std::shared_ptr<typename IfcEntityTypesT::IfcBoundedSurface>& splineSurface,
-						const std::vector<std::vector<carve::geom::vector<3>>>& controlPoints,
+						const EXPRESSReference<typename IfcEntityTypesT::IfcBSplineSurface>& splineSurface,
+						const carve::math::Matrix& pos, // AT THE MOMENT, NO IDEA WHAT THIS IS

```

Comment:
`pos` is the position of the geometry in the global context (is specified elsewhere in the IFC file and thus provided as a parameter to the function)  and should be accounted for when interpreting the geometries. This is a parameter in almost all functions.

Reply:
Done in commit c627ad8.